data/wicked/arp-tool.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+import scapy.all as scapy
+import ipaddress
+import time
+import argparse
+import sys
+
+
+def defend(interface, network, count):
+    myMAC = scapy.get_if_hwaddr(interface)
+    counter = 0
+    ipaddress.ip_network(network)
+
+    def handle_packet(packet, ctx):
+        if packet[scapy.ARP].op != 1: 
+            return
+
+        if ipaddress.ip_address(packet.pdst) not in ipaddress.ip_network(ctx['net']):
+            print("IGN: Ignore incomming arp request with pdst=" + packet.pdst)
+            return
+
+        if str(packet.src) in ctx['ignore_src']:
+            print("IGN: Ignore incomming arp request by MAC with src=" + packet.src)
+            return
+
+        print("RCV: " + packet.__repr__())
+        reply = scapy.ARP(op=2,  hwsrc=myMAC, psrc=packet.pdst, hwdst=packet[scapy.ARP].hwsrc, pdst="0.0.0.0")
+        reply = scapy.Ether(dst="ff:ff:ff:ff:ff:ff", src=myMAC) / reply
+
+        scapy.sendp(reply, iface=ctx['ifc'], verbose=False)
+        print("SND: " + reply.__repr__())
+        print("");
+
+        ctx['pkt_count'] = ctx['pkt_count'] + 1
+        if(ctx['max_count'] > 0 and ctx['pkt_count'] >= ctx['max_count']):
+                sys.exit(0)
+        return
+
+    my_macs=list(filter(lambda x: x != '00:00:00:00:00:00', [scapy.get_if_hwaddr(i) for i in scapy.get_if_list()]))
```

Comment:
```suggestion
    my_macs = {scapy.get_if_hwaddr(i) for i in scapy.get_if_list()} ^ {'00:00:00:00:00:00'}
```

Reply:
It's just sets. I tried in my machine and it works. XD

Reply:
And I like it, thx for the pointer!